src/Raven.Server/Documents/PeriodicBackup/RavenFtpClient.cs

```diff
@@ -53,6 +60,7 @@ public RavenFtpClient(FtpSettings ftpSettings, Progress progress = null, Cancell
                 _url += "/";
 
             Debug.Assert(_url.StartsWith("ftp://", StringComparison.OrdinalIgnoreCase));
+            _isTesting = Environment.GetEnvironmentVariable("isTesting") != null;
```

Comment:
not a good idea, would prefer a local field or a static set in the TestBase or RavenTestBase (in static ctor)

Reply:
Done